emscripten.py

```diff
@@ -1539,6 +1543,11 @@ def math_fix(g):
      'Module' + access_quote('asmLibraryArg'),
      receiving)]
 
+  indirect_funcs = '[' + ','.join(['"%s"' % asmjs_mangle(f) for f in metadata['indirect_table']]) + ']'
```

Comment:
I'm not exactly sure how the quoting in these lines needs to look, plz take a look at this in particular.